docs/java-rest/high-level/document/exists.asciidoc

```diff
@@ -0,0 +1,59 @@
+[[java-rest-high-document-exists]]
+=== Exists API
+
+[[java-rest-high-document-exists-request]]
+==== Exists Request
+
+The `exists()` API is `true` if a document exists, and `false` otherwise.
+It uses `GetRequest` just like the <<java-rest-high-document-get>>.
+All of its <<java-rest-high-document-get-request-optional-arguments, optional arguments>>
+are supported. Since `exists()` only returns `true` or false, we recommend
+turning off fetching `_source` and any stored fields so the request is
+slightly lighter:
+
+["source","java",subs="attributes,callouts,macros"]
+--------------------------------------------------
+include-tagged::{doc-tests}/CRUDDocumentationIT.java[exists-request]
+--------------------------------------------------
+<1> Index
+<2> Type
+<3> Document id
+<4> Disable fetching `_source`.
+<5> Disable fetching stored fields.
```

Comment:
Hm... I would say with `.` or without `.` for all? What do you think?

Reply:
I put the `.` because these are sentences and the ones above aren't.